renku/core/utils/git.py

```diff
@@ -68,3 +70,16 @@ def split_paths(*paths):
 
     for index in range(batch_count):
         yield paths[index * ARGUMENT_BATCH_SIZE : (index + 1) * ARGUMENT_BATCH_SIZE]
+
+
+def get_slug(name):
```

Comment:
`renku.core.utils.git` might be a better place for this, as a "slug" is an URL concept.